pkgs/applications/misc/ksmoothdock/default.nix

```diff
@@ -25,6 +25,7 @@ mkDerivation rec {
       url = "https://github.com/dangvd/ksmoothdock/commit/00799bef8a1c1fe61ef9274866267d9fe9194041.patch";
       sha256 = "1nmb7gf1ggzicxz8k4fd67xhwjy404myqzjpgjym66wqxm0arni4";
     })
+    ./qt515.patch
```

Comment:
Where does this patch come from? Please use `git format-patch`.

Reply:
I wrote it

Reply:
Maybe contribute it upstream? Even if they won't accept it immediately, you can fetch it here.

Reply:
Done https://github.com/dangvd/ksmoothdock/pull/123

Reply:
Great :) Now it would be great if you'd use `fetchpatch` and add a link in a comment to the PR. Also, use the following link:

https://github.com/dangvd/ksmoothdock/commit/1f4c94e289f078bdaba7bb165c5cf2a9f743076e.patch